corehq/apps/api/user_updates.py

```diff
@@ -42,7 +43,12 @@ def update(user, field, value, user_change_logger=None):
 
 
 def _update_email(user, email, user_change_logger):
-    _simple_update(user, 'email', email.lower(), user_change_logger)
+    email = email.lower()
+    try:
+        validate_email(email)
+    except ValidationError:
+        raise UpdateUserException(_("The value '{}' for 'email' must be a valid email address").format(email))
```

Comment:
Nit: (feel free to disregard) `the value <val> for 'email' must be a valid` feels a little awkward to me.
```suggestion
        raise UpdateUserException(_("Invalid 'email' field (must be a valid email address): {!r}").format(email))
```